src/core/reference.js

```diff
@@ -40,6 +57,14 @@
  */
 
 /**
+ * The <a href="#/p5/if-else">if-else</a> statement helps control the flow of your code.
```

Comment:
Should the comparison operators be added? Since they are key for if and for? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators similarly, should examples in general use double or triple equals?

Reply:
Added comparison operator examples. The linter requires the triple equals in all examples see comment below for more.